### PR TITLE
HJB example code and dependency install file.

### PR DESCRIPTION
**Pull request by zzj0402**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,15 +7,21 @@
...
+neural networks (PINNs) and deep BSDE solvers. It utilizes techniques like deep neural networks and neural
 stochastic differential equations to make it practical to solve high dimensional PDEs efficiently through the
 likes of scientific machine learning (SciML).
 
-# Examples
+## Installation
 
-## DeepBSDE Solver
+```bash
+julia ./install.jl
```
Comment (ChrisRackauckas) [inline README.md:17]: This is very non-standard and we should stick to standard techniques, e.g. using the Pkg manager REPL (Also, this doesn't work on all operating systems)

**Comment by ChrisRackauckas** [comment]
This should probably move towards populating and deploying the documentation?

https://github.com/SciML/NeuralNetDiffEq.jl/tree/master/docs/src

I'm not sure the README should get much work knowing it will get deleted soon.

**Comment by ChrisRackauckas** [comment]
See https://github.com/SciML/NeuralNetDiffEq.jl/issues/85 for details.

File: README.md (hunk 1 above)
Comment (kanav99) [reply README.md:17]: You can make a new environment for `examples` folder and add the dependencies to the Project.toml

Comment (ChrisRackauckas) [reply README.md:17]: We should probably just have a tutorials section of the documentation like is done in DiffEqFlux, DataDrivenDiffEq, etc.

Comment (zzj0402) [reply README.md:17]: Okay, working it.

**Pull request closed without merging**